### ISSUE TITLE
String 6646–6647

### ISSUE THREAD

**Issue by AaronVanGeffen** [OpenRCT2/Localisation#2868]
New strings from PR OpenRCT2/OpenRCT2#20952.

```
STR_6646    :Author: {STRING}
STR_6647    :Authors: {STRING}
```

Updated in:
- [ ] ar-EG
- [ ] ca-ES
- [ ] cs-CZ
- [ ] da-DK
- [ ] de-DE
- [x] en-US
- [ ] eo-ZZ
- [ ] es-ES
- [ ] fi-FI
- [ ] fr-FR
- [ ] hu-HU
- [ ] it-IT
- [ ] ja-JP
- [ ] ko-KR
- [ ] nb-NO
- [ ] nl-NL
- [ ] pl-PL
- [ ] pt-BR
- [ ] sv-SE
- [ ] tr-TR
- [ ] uk-UA
- [ ] zh-CN
- [ ] zh-TW

Notification when new strings arrive:
@OmarAglan @J0anJosep @LPSGizmo @Wuzzy2 @gr33ndev @tellovishous @dimateos @groenroos @rmnvgr @LucaRed @telk5093 @Mmmmulder @Gymnasiast @marcinkunert @Tupaschoal @CsyeCokTheSolly @LPR-Michael @mrmagic2020 @daihakken

**Comment by Gymnasiast** [comment]
I think we already had these, as they were added for the Scenery window.

**Comment by AaronVanGeffen** [comment]
Ahh, you're right, there's a `STR_SCENERY_AUTHOR` and `STR_SCENERY_AUTHOR_PLURAL` as well. They can indeed be unified, as far as I'm concerned.

**Comment by tupaschoal** [comment]
I thought we always duplicated strings

**Comment by groenroos** [comment]
Yes, please do always duplicate strings! It's unknowable how different languages may require different grammar for the "same" label in different contexts.
